components/google-cloud/RELEASE.md

```diff
@@ -1,5 +1,6 @@
 ## Upcoming release
 * Remove default prediction column names in `v1.model_evaluation.regression_component` component to fix pipeline errors when using bigquery data source.
+* Add a new `use_venv` field to the component decorator, enabling the component to run inside a virtual environment.
```

Comment:
Would be nice to document the use of the `use_venv` field in the component decorator to ensure users understand when and how to use it.

Reply:
Do you mean this https://github.com/kubeflow/pipelines/pull/11326/files#diff-6a9e82b5686376e314dddd14a9faefcecf870d0b42cb2256d3934f7fbf2031a4R79?